config.example.php

```diff
@@ -43,7 +43,7 @@
         ],
         'ftp' => [
             'plain' => false,       // set true to use FTP instead of FTPS e.g. on Windows
-            'enabled' => true,      // set true if your FRITZ!Box does not support ftp - e.g. 7412
+            'enabled' => true,      // set false if your FRITZ!Box does not support ftp - e.g. 7412
```

Comment:
Das Problem bleibt- in alter Config ist der Key nicht drin! Nenn ihn doch einfach `disable`, dann macht es auch Sinn auch "true" zu prüfen. Es muss trotzdem sichergestellt sein, dass das auch funktioniert wenn es den key nicht gibt.

Reply:
Äh, ich versteh  nicht, was du mit "muss trotzdem sichergestellt sein, dass das auch funktioniert wenn es den key nicht gibt." meinst. Ich mach ja alles was du willst, aber dein Vorschlag war doch,
statt;
```PHP
'plainFTP' => false,
'ftp-disable' => false,
```
```PHP
'ftp' => [
    'plain' => false,
    'enabled' => true,
]
```
zu nutzen.

Reply:
Jetzt habe ich es noch einmal auf links gedreht und mit
```PHP
'disable' => false,
```
und
```PHP
'disable' => true,
```
getestet.
Wenn das Flag fehlt (auskommentiert ist) geht das script so oder so auf die Bretter (keine Abwärtskompatibilität zu vorheriger config).
Der Standardfall (so in der config.example.php) muss `'disable' => false,` sein, da ja nur im Ausnahmefall ftp ausgeschaltet wird.